firmware-nucleo/smart-air-hockey-table/App/app_statemachine.c

```diff
@@ -0,0 +1,77 @@
+#include "app_statemachine.h"
+
+static volatile GameState currGameState;
+// TODO: Make winScore modifiable by the player through the OLED menu, for now it is hard-coded to 7
+static volatile uint8_t winScore = 7; // Number of points required for a player to win
+static volatile uint8_t playerScores[NUM_PLAYERS];

```

Comment:
I don't think I like scores being in an array

Reply:
Switched from a "player index" to a "player ID" in b009e5d